### DISCUSSION TITLE
Wrong ephemerides of Jupiter's Moon Himalia?

### DISCUSSION THREAD

**Discussion by Mario-Helm** [Stellarium/stellarium#3494] [General]
Hello to all, nice to be in this forum. I have a problem with taking photographs of Jupiter's moon Himalia. After many trials without success I found out that my desktop version of Stellarium shows obviously a wrong position of this moon. It is not a small deviation - it is indeed on the wrong side of Jupiter (actually 3m in RA and 1 degree deviation in DEC!). The web and Android versions show the moon on the right side of Jupiter in a position which at least nearly coincides with the ephemeride from https://www.minorplanetcenter.net/cgi-bin/natsats.cgi, but also not exactly in the same position.  However, also the DEC position of Jupiter itself and other objects differ by some arc minutes, if I compare the web version, my Android version and the desktop version at at (hopefully) same time and location. But here at least the whole configuration is as expected. Has anybody also found a similar problem and any solution for me? Thanks a lot for your help :-).

**Comment by gzotti** [comment]
An error in the position of Jupiter would be surprising. Himalia and several other smaller moons use very simple orbit models with possibly outdated orbital elements. Professional moon observers might prefer e.g. JPL Horizons for such hunts.
